CMSgov/mint-app
# [EASI-2218] Read-only Model sub-page: Team information - FE

# EASI-2218

## Changes and Description

- Creates new Team Info Read Only component
- Adds `teamRole` field to query to `src/queries/ReadOnly/GetModelSummary.ts`
- Update `ContactInfo` Side Navigation component to only show Model Leads 

## Acceptance Criteria

- Reusable component for rendering collaborator info
- Testing


## How to test this change

1. Navigate to the Model Summary ( Models on nav > Complete Plan > View this Model Plan on sidebar)
2. Click `Team` on sidebar
3. Ensure component is displayed correctly and matches figma

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
